openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -189,27 +194,37 @@ def get_cohort(user, course_key, assign=True):
     Raises:
        ValueError if the CourseKey doesn't exist.
     """
+    request_cache = RequestCache.get_request_cache()
+    cache_key = u"user_{}_cohort".format(user.id)
```

Comment:
1. Doesn't the course need to be part of this key?
2. Can the cache_key have a prefix that makes it more obvious where it's coming from (e.g. `cohorts.get_cohort....`)